pkg/util/ssh.go

```diff
@@ -142,19 +142,19 @@ func (s *SSHTunnel) Close() error {
 	return nil
 }
 
-func RunSSHCommand(cmd, host string, signer ssh.Signer) (string, string, int, error) {
+func RunSSHCommand(cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
```

Comment:
Sure, but not obvious to me how to do that.  I could create and pass in an SSH dialer, which would wrap ssh.Dial and verify the host & config.  Is that what you had in mind?